src/Helper.ts

```diff
@@ -232,64 +228,35 @@ class Helper {
      *
      * @type {string[]}
      */
-    const states = [];
+    const states: string[] = [];
 
     if (!this.isInitialized()) {
       console.warn("Helper class should be initialized before calling this method!");
     }
 
     // Get the ID of the devices which are linked to the given area.
     for (const area of this.#areas) {
-      const areaDeviceIds = this.#devices.filter(device => {
+      const areaDeviceIds = this.#devices.filter((device: any) => {
         return device.area_id === area.area_id;
-      }).map(device => {
+      }).map((device: generic.deviceEntity) => {
         return device.id;
       });
 
       // Get the entities of which all conditions of the callback function are met. @see areaFilterCallback.
       const newStates = this.#entities.filter(
-          this.#areaFilterCallback, {
-            area: area,
-            domain: domain,
-            areaDeviceIds: areaDeviceIds,
-          })
-          .map(entity => `states['${entity.entity_id}']`);
+        this.#areaFilterCallback, {
+          area: area,
+          domain: domain,
+          areaDeviceIds: areaDeviceIds,
+        })
+        .map((entity: any) => `states['${entity.entity_id}']`);
```

Comment:
```suggestion
        .map((entity: generic.hassEntity) => `states['${entity.entity_id}']`);
```

Reply:
Good catch, but you're way ahead of me... The type was set to `any` to be able to compile the script to test it for functionality. All `any` types must still be evaluated.
In this case I've changed the type already locally.